### DISCUSSION TITLE
How can the current price be queried?

### DISCUSSION THREAD

**Discussion by realhik** [BeatsuDev/tibber.py#25] [Q&A] [answered]
Can you give an example how the current price can be queried?

**Comment by ericbstie** [comment, marked as answer]
Hi! Thanks for creating a discussion. I think this should do the trick:
```py
import tibber

account = tibber.Account("<tibber token>")

home = account.homes[0]  # Choose a home
subscription = home.subscriptions[0]  # Choose a subscription
price = subscription.price_info.current
```
The price variable is of [type `Price`](https://github.com/BeatsuDev/tibber.py/blob/fbd4dcea54c578d644a0b9ecc94ed9d72d77997f/tibber/types/price.py#L9), so you can get the total price, tax, energy price without tax etc. like this:
```py
print(price.total)
print(price.tax)
print(price.energy)
print(price.currency)
```
